lib/algora_web/components/core_components.ex

```diff
@@ -767,6 +767,29 @@ defmodule AlgoraWeb.CoreComponents do
     """
   end
 
+  def input(%{type: "radio", value: value} = assigns) do
```

Comment:
I had to implement a radio button since they aren't yet being used. I think it's the semantically correct element for acessibility, keynoard nav etc but it's only been shallowly styled 🤷🏻

Reply:
Yeah radio button works great! :+1: Re: styling, can we make the inputs `sr-only`, reduce the opacity of inactive thumbnails and style the active one with a white border?

Reply:
Something along the lines of
![](https://github.com/user-attachments/assets/9c0caa59-1a75-4992-b830-778ca368f711)

Reply:
Have done so

![image](https://github.com/user-attachments/assets/25c3f995-da8d-4354-89c5-d2353273bbf7)